.github/workflows/release.yml

```diff
@@ -263,3 +263,13 @@ jobs:
         with:
           name: tari_universe.pdb
           path: "${{ github.workspace }}/src-tauri/target/release/tari_universe.pdb"
+
+      - name: Windows debug symbols - Upload to Sentry
+        if: startsWith(runner.os,'Windows')
+        env:
+          AUTH_TOKEN: ${{ secrets.SENTRY_SYMBOLS_AUTH_TOKEN }}
+        continue-on-error: true
+        shell: bash
+        run: |
+          npm install @sentry/cli
+          sentry-cli debug-files upload --org tari-labs --project tari-universe ${{ github.workspace }}/src-tauri/target/release
```

Comment:
Think we need to add the actual symbols file
```bash
sentry-cli debug-files upload --org tari-labs --project tari-universe ${{ github.workspace }}/src-tauri/target/release/tari_universe.pdb
```

Reply:
I'll try this. But my testing showed it wanted a dir to scan, not a file to upload.

Reply:
I tried it. You are correct it works :D